sources/editor/Stride.Core.Assets.Editor/View/DefaultPropertyTemplateProviders.xaml

```diff
@@ -157,9 +157,9 @@
         <!-- Parameters of the SumNum converters used here must be the total width of fixed columns + MinWidths of sizable columns: 8+5+64=77 -->
         <ColumnDefinition Width="8"/>
         <ColumnDefinition Width="{Binding NameColumnSize, RelativeSource={RelativeSource FindAncestor, AncestorType={x:Type sd:PropertyView}}, Mode=TwoWay}"
-                          MaxWidth="{Binding ActualWidth, ElementName=Grid, Converter={sd:SumNum}, ConverterParameter={sd:Double -77}}" MinWidth="64"/>
+                          MaxWidth="{Binding ActualWidth, ElementName=Grid, Converter={sd:SumNum}, ConverterParameter={sd:Double 77}}" MinWidth="64"/>
```

Comment:
Setting the value to a positive one doesn't make more sense: we would be expanding the MaxWidth to an ever increasing value. After some thought I believe it might have been a workaround for some constraints not working on columns in an earlier version of WPF. Today, it is pointless. A final fix should be to just remove it altogether.

```suggestion
                          MinWidth="64"/>
```

And apply the same on all other cases (as well as removing the comment above).